src/ceph.in

```diff
@@ -552,8 +552,8 @@ def ping_monitor(cluster_handle, name, timeout):
         return 1
 
     mon_id = name[len('mon.'):]
+    run_in_thread(cluster_handle.connect, timeout=timeout)
```

Comment:
i think it's fine if `ceph ping mon.a` does not connect to the cluster before hand. the reason that `ceph ping mon.*` needs to connect to cluster is that it needs to get the monitor list using "ceph mon dump" command, before pinging them one by one.

but in the case of `ceph ping mon.a`, instead of using the output of "ceph mon dump", MonClient will try to look up the monitor id in the initial monmap. the initial monmap is built from the `ceph.conf`.